ngraph/core/src/op/deformable_psroi_pooling.cpp

```diff
@@ -81,13 +81,13 @@ void op::v1::DeformablePSROIPooling::validate_and_infer_types()
 
     NODE_VALIDATION_CHECK(this,
                           input_pshape.rank().is_dynamic() || input_pshape.rank().get_length() == 4,
```

Comment:
This condition could be simpler expressed as:
`input_pshape.rank().compatible(4)`

Reply:
Done